src/handlers/oauth_handler.js

```diff
@@ -230,58 +208,60 @@ async function getPat(req, res) {
   }
 
   try {
-    const user_data = await superagent
+    const userData = await superagent
       .get("https://api.github.com/user")
       .set({ Authorization: `Bearer ${params.token}` })
       .set({ "User-Agent": GH_USERAGENT });
 
-    if (user_data.status !== 200) {
+    if (userData.status !== 200) {
       logger.generic(2, "User Data Request to GitHub Failed!", {
         type: "object",
-        obj: user_data,
+        obj: userData,
       });
       await common.handleError(req, res, {
         ok: false,
         short: "Server Error",
-        content: user_data,
+        content: userData,
       });
       return;
     }
 
     // Now to build a valid user object
-    let userObj = {
-      username: user_data.body.login,
-      node_id: user_data.body.node_id,
-      avatar: user_data.body.avatar_url,
-    };
+    const username = userData.body.login;
+    const userId = userData.body.node_id;
+    const userAvatar = userData.body.avatar_url;
 
-    let check_user_existance = await database.getUserByNodeID(userObj.node_id);
+    const userExists = await database.getUserByNodeID(userId);
 
-    if (check_user_existance.ok) {
-      logger.generic(4, `User Check Says User Exists: ${userObj.username}`);
+    if (userExists.ok) {
+      logger.generic(4, `User Check Says User Exists: ${username}`);
 
       // If we plan to allow updating the user name or image, we would do so here
-      userObj.token = params.token;
 
       // Now we redirect to the frontend site.
-      res.redirect(`https://web.pulsar-edit.dev/users?token=${userObj.token}`);
+      res.redirect(`https://web.pulsar-edit.dev/users?token=${params.token}`);
       logger.httpLog(req, res);
       return;
     }
 
-    let create_user = await database.insertNewUser(userObj);
+    let createdUser = await database.insertNewUser(username, userId, userAvatar);
 
-    if (!create_user.ok) {
-      logger.generic(2, `Creating User Failed! ${userObj.username}`);
-      await common.handleError(req, res, create_user);
+    if (!createdUser.ok) {
+      logger.generic(2, `Creating User Failed! ${username}`);
+      await common.handleError(req, res, createdUser);
       return;
     }
 
-    create_user.content.token = params.token;
+    // TODO: For now we will just return the user data as JSON.
```

Comment:
I appreciate that you are adding this from the previous identical handling of OAuth tokens, but we are actually good to remove this comment in both locations.

Since we do now redirect both requests to the frontend site, the comment is outdated in both locations